.github/workflows/release.yml

```diff
@@ -0,0 +1,158 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+env:
+  RELEASE_BRANCH: master
+  RELEASE_CANDIDATE_TAG_PATTERN: ^v[0-9]+.[0-9]+.[0-9]+-rc
+  RELEASE_SIGNING_KEY_ID: 75B0 21D9 64AD 8693 24CD  B74D DDD7 BCCD 44BB 9A9D
```

Comment:
Is this your key? Might be worth adding the ability to receive multiple keys and verifying that one of the keys signed the commit, that way you and I can individually sign release tags if needed without sharing some common key.

Reply:
This is your key. We can add mine too later if that's okay with you.